azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_transport.py

```diff
@@ -203,6 +217,39 @@ def _on_provider_publish_complete(self, mid):
             del self._event_callback_map[mid]
             callback()
 
+    def _on_provider_subscribe_complete(self, mid):
+        """
+        Callback that is called by the provider when a subscribe operation is complete.
+        """
+        logger.info("_on_provider_subscribe_complete")
+        callback = self._subscribe_callback
```

Comment:
>_subscribe_callback [](start = 24, length = 19)

isn't that going to be overwritten if we have multiple subscriptions going on at the same time?